Pinta.Core/Managers/LivePreviewManager.cs

```diff
@@ -130,12 +130,12 @@ public void Start (BaseEffect effect)
 		if (!effect.IsTileable)
 			tileHeight = render_bounds.Height;
 
-		var settings = new AsyncEffectRenderer.Settings () {
-			ThreadCount = system_manager.RenderThreads,
-			TileWidth = tileWidth,
-			TileHeight = tileHeight,
-			ThreadPriority = ThreadPriority.BelowNormal
-		};
+		AsyncEffectRenderer.Settings settings = new (
+			threadCount: system_manager.RenderThreads,
+			tileWidth: tileWidth,
+			tileHeight: tileHeight,
+			updateMilliseconds: 100,
+			threadPriority: ThreadPriority.Normal);
```

Comment:
Was there a reason for changing the thread priority?

Reply:
@cameronwhite no, this was a mistake on my part when using autocomplete in my IDE. I've corrected it now